src/cunit/ut_cunit_internal.h

```diff
@@ -51,7 +51,7 @@ extern void UT_console_run_tests( void );
 extern void UT_automated_run_tests(void);
 
 extern void UT_list_tests(CU_pSuite pSuite);
-void UT_set_output_filename(const char* szFilenameRoot);
+void UT_set_results_output_filename(const char* szFilenameRoot);
```

Comment:
ut_cuinit -> is specific to cunit, but what we're actually saying for this function, is that it must exist on all frameworks, because we call it from ut_main.

Therefore it's an internal function which is implemented by the framework.

so ut_interna.h is the correct location, I've move this and another function we're calling into the ut_internal.h in your local directory just to check I was correct, can you commit that?.

Reply:
sure, done.